tests/phpunit/integration/tests/Media/Video/Muting.php

```diff
@@ -210,8 +225,10 @@ public function test_delete_video_meta_attachment_is_deleted(): void {
 			]
 		);
 
+		$this->assertNotWPError( $muted_attachment_id );
+
 		add_post_meta( $video_attachment_id, $this->instance::MUTED_ID_POST_META_KEY, $muted_attachment_id );
-		$this->assertSame( $muted_attachment_id, (int) get_post_meta( $video_attachment_id, $this->instance::MUTED_ID_POST_META_KEY, true ) );
+		$this->assertSame( $muted_attachment_id, get_post_meta( $video_attachment_id, $this->instance::MUTED_ID_POST_META_KEY, true ) );
```

Comment:
Looks like we need to add back the casting of int.

Reply:
You cannot cast mixed to int, so it would need to be with a phpdoc block